frb_codegen/src/generator/dart/ty.rs

```diff
@@ -1,6 +1,12 @@
+use std::{collections::HashMap, sync::Mutex};
+
 use crate::{generator::dart::*, Opts};
 use enum_dispatch::enum_dispatch;
 
+lazy_static! {
+    pub static ref REQUIRES_VALIDATION: Mutex<HashMap<String, bool>> = Mutex::new(HashMap::new());
```

Comment:
uber-nit: Where is it used? Maybe we can try to avoid static variables since they may not be perfect programming styles

Reply:
forgot to delete